_apps/rtd-bot.md

```diff
@@ -0,0 +1,20 @@
+---
+# A human-friendly name of your listing
+title: Read The Docs
+# A short description of what your app does
+description: An unofficial GitHub Probot that helps reveiwing on Read The Docs
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: rtd-bot
```

Comment:
Any chance you'd consider renaming to a slug with `bot` in it? The GitHub UI already displays `[bot]` next to every GitHub App.

Reply:
It seems that `rtd` is already in use. And I don't want to use `Read The Docs` that may be used by official one in future (they use this name for [GitHub Services](https://github.com/github/github-services/blob/a71f251994d146e43843b9e6659da87ecdb03a58/lib/services/read_the_docs.rb)).

Is it OK to keep this name?